CADETProcess/simulator/cadetAdapter.py

```diff
@@ -1398,7 +1398,9 @@ class UnitParameters(ParameterWrapper):
 adsorption_parameters_map = {
     'NoBinding': {
         'name': 'NONE',
-        'parameters': {},
```

Comment:
Why does `NoBinding` now have the `is_kinetic` parameter?

Reply:
Because the NoBinding class inherits from BindingBaseClass, and BindingBaseClass has is_kinetic. And my tests tested if every attribute from the class is present in the adapter and flagged NoBinding because it didn't have is_kinetic.

Reply:
Ah, I understand. That's actually not super clean architecture since `NoBinding` obviously does not have any kinetics associated.
Nonetheless, I'm surprised because with the current `Structure` design, classes should automatically inherit all parameters from the parent. And in my local test (on `dev`) this is also the case:

```
foo = NoBinding()

print(foo.parameters)
>>> {'is_kinetic': True}
```

I'll have to think about it.

Reply:
Sorry, now I understand. It DOES have the parameter and that's the issue!

Ok, still have to think about it! :sweat_smile:

Reply:
We could remove the is_kinetic attribute from the NoBinding. That'd be cleaner. Should I push that?

Reply:
No, thanks. I think it's fine for now. It won't do harm since CADET-Core ignores superfluous parameters anyway.

Thanks for the PR, it looks good to me. I will merge it.